### ISSUE TITLE
Remove hard-coded port values

### ISSUE THREAD

**Issue by mrlubos** [sofn-xyz/mailing#117]
While we can configure the port for our mailing application by using the `--port` CLI flag, searching for "3883" still returns 12 results in 8 files. At least a few of these instances should be replaced with the port value from argument, otherwise the application gets noisy in the console. This is what happens if I execute `mailing preview --port=8080`

```sh
[mailing] Watching for changes to emails
[mailing] Running preview at http://localhost:8080/
[mailing] 200 / 57ms
[mailing] 200 /previews.json 3831ms
[mailing] 200 /logo-small@2x.png 80ms
FetchError: request to http://localhost:3883/previews/file.tsx/template.json failed, reason: connect ECONNREFUSED 127.0.0.1:3883
    at ClientRequest.<anonymous> (/root/node_modules/next/dist/compiled/node-fetch/index.js:1:65763)
    at ClientRequest.emit (node:events:526:28)
    at ClientRequest.emit (node:domain:475:12)
    at Socket.socketErrorListener (node:_http_client:442:9)
    at Socket.emit (node:events:526:28)
    at Socket.emit (node:domain:475:12)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  type: 'system',
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED'
}
```

**Comment by psugihara** [comment]
This was fixed in #131

**Comment by mrlubos** [comment]
Thanks as always @psugihara!
